data/gegno/power.txt

```diff
@@ -0,0 +1,94 @@
+# Copyright (c) 2021 by Saugia
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+outfit "R01 Skirmish Battery"
+	category Power
+	plural "R01 Skirmish Batteries"
+	mass 12
+	thumbnail outfit/gbatterytiny
+	"energy capacity" 1121
+	"outfit space" -12
+	description "Skirmish batteries are very common amongst ships across Gegno space. They are a perfectly convenient storage of battery power to offset the load of larger Gegno generators, or a nice addition to add on top of them where more energy would be needed."
+
+outfit "R02 Battlezone Battery"
+	category Power
+	plural "R02 Battlezone Batteries"
+	mass 23
+	thumbnail outfit/gbatterysmall
+	"energy capacity" 2237
+	"outfit space" -23
+	description "Battlezone batteries are primarily used on the Gegno's medium-range combat vessels, holding a decent amount of power for long lasting engagements. They also fit nicely on larger vessels as an extra power source to feed their larger systems."
+
+outfit "R03 Warforge Battery"
+	category Power
+	plural "R03 Warforge Batteries"
+	mass 41
+	thumbnail outfit/gbatterymed
+	"energy capacity" 4469
+	"outfit space" -41
+	description "Warforge batteries have proved their usefulness over the years. As large, sturdy battery packs, they assist with equally large vollies that are fired from Gegno ships, and help maintain an excess of power for engine usage."
+
+outfit "R04 Crusade Battery"
+	category Power
+	plural "R04 Crusade Batteries"
+	mass 83
+	thumbnail outfit/gbatterylarge
+	"energy capacity" 8885
+	"outfit space" -83
+	description "As the largest battery mounted on Gegno ships, the Crusade Battery supplies almost enough power as a mini generator. They are enough to support the massive warships of both the Scin and the Vi. However, they are slightly under the capacity of similar human batteries due to inefficient handling of their scale."
+
+outfit "C3 Scrap Cell"
+	category Power
+	mass 25
+	thumbnail outfit/vicoretiny
+	"energy generation" 1.4
+	"heat generation" 2.2
+	"outfit space" -25
+	description "The Scrap Cell is a small radiothermal generator that produces energy generation in a pinch, making use of the Gegno's slightly hotter running tech. Its output is very limited, and is only used in tandem with the Gegno's smaller-sized systems."
```

Comment:
Another description that I need to edit more intuitively- it's basically the reasoning behind making it a radio _thermal_ generator- Gegno ships default to running hot without their systems, so the "thermal" part ties in with that. This will make a bit more sense once I introduce the shield-alternative-hull outfit though.